examples/sanitizers/.bazelrc

```diff
@@ -3,3 +3,6 @@ import %workspace%/../../shared.bazelrc
 
 # Use a user.bazelrc if it exists
 try-import %workspace%/user.bazelrc
+
+# Build for iOS platform
+build --apple_platform_type=ios
```

Comment:
This shouldn't be needed?

Reply:
Oh yes, I wanted to get your thoughts on this. Its not needed for the Xcode experience but `bazel build ...` inside the `examples/sanitizers` workspace failed with this error:
```
UndefinedBehaviorSanitizerApp/Sources/main.m:1:9: fatal error: 'UIKit/UIKit.h' file not found
#import <UIKit/UIKit.h>
```
This further failed the CI pipeline that is linked to the PR checks, hence this was added.

Reply:
Adding a `"manual"` tag to the `objc_library` targets should fix it.

Reply:
Oh yeah :) I will update it right away...Thanks!